docs/contributing/style_guide.md

```diff
@@ -198,9 +197,9 @@ export type { Person } from "./my_file.ts";
 
 ### Minimize dependencies; do not make circular imports.
 
-Although `cli/js` and `std` have no external dependencies, we must still be
-careful to keep internal dependencies simple and manageable. In particular, be
-careful not to introduce circular imports.
+Although `std` has no external dependencies, we must still be careful to keep
```

Comment:
runtime/js would be where most of it is now.

Reply:
i mostly removed it since this is under the `Typescript` section, but a possible solution that would still include `runtime/js` would be to convert it to `Typescript/Javascript` and include the parts with `runtime/js` again, not sure if it would cause confusion around typing though